src/ophys_etl/transforms/classification.py

```diff
@@ -331,11 +331,19 @@ def load_model(classifier_model_uri: str) -> Any:
     return model
 
 
-def main(parser):
-    with open(parser.args["roi_masks_path"], "r") as f:
+def filtered_roi_load(roi_masks_path):
+    with open(roi_masks_path, "r") as f:
         raw_roi_data = json.load(f)
     roi_input_schema = SparseAndDenseROISchema(many=True)
     roi_data = roi_input_schema.load(raw_roi_data)
+    excluded = [r for r in roi_data if r["exclusion_labels"]]
```

Comment:
I think it should be?

`excluded = [r for r in roi_data if any(r["exclusion_labels"])]`

Same for Line 340 as well.

Reply:
It is implicit that `if []` evaluates as `if False` and `if [anything]` evaluates as `if True`

Reply:
Right! Haven't had my morning coffee yet.